src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -131,14 +133,27 @@ class HttpGoogleServicesDAO(
     def insertBucket: (Member) => Future[Unit] = { _ =>
       retry(when500) {
         () => Future {
-          val bucketAcls = groupAccessLevelsAscending.map(ac => newBucketAccessControl(makeGroupEntityString(toGroupId(bucketName, ac)), ac))
+          // bucket ACLs should be:
+          //   workspace owner - bucket writer
+          //   workspace writer - bucket writer
+          //   workspace reader - bucket reader
+          //   bucket service account - bucket owner
+          val workspaceAccessToBucketAcl = Map(Owner -> "WRITER", Write -> "WRITER", Read -> "READER")
+          val bucketAcls =
```

Comment:
Format `bucketACLs` and `defaultObjectAcls` lines in the same way, to make their similarity more obvious on first glance